modules/apps/headless/headless-asset-library/headless-asset-library-impl/rest-openapi.yaml

```diff
@@ -466,10 +466,41 @@ paths:
                   schema:
                       type: string
             responses:
-                204:
+                200:
                     content:
-                        application/json: {}
-                        application/xml: {}
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/UserGroup"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/UserGroup"
+                    description:
+                        default response
+            tags: ["UserGroup"]
+        put:
+            description:
+                Associates the user group the given external reference code to the asset library.
```

Comment:
missing "with" in "the user group the given external"

Reply:
Fixed.